Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
+ENV PATH $PATH:/usr/local/bin
+ENV TERRAFORM_VER 0.6.7
+ENV TERRAFORM_ZIP terraform_"$TERRAFORM_VER"_linux_amd64.zip
+
+RUN apt-get update && apt-get -y install wget unzip openssh-client dnsutils git pass ruby
```

Comment:
> Do we need pass here?

It depends how we want to pass CF credentials. 

Other packages:
- wget - downloads terraform
- unzip - unzips terraform
- ssh - terraform uses it to upload files
- dnsutils - debugging (nslookup/dig etc.)
- git - clone pass repo
- ruby - bosh